optuna/samplers/_tpe/parzen_estimator.py

```diff
@@ -190,7 +189,8 @@ def _calculate_categorical_distributions(
                 weights=np.full((1, n_choices), fill_value=1.0 / n_choices)
             )
 
-        n_kernels = len(observations) + parameters.consider_prior
+        consider_prior = True
+        n_kernels = len(observations) + consider_prior
```

Comment:
[nit] An inline comment is sufficient.
```suggestion
        n_kernels = len(observations) + 1  # NOTE(sawa): +1 for prior.
```